### DISCUSSION TITLE
Announcement: New book on ESP32 and MicroPython

### DISCUSSION THREAD

**Discussion by simonmonk** [micropython/micropython#15255] [ESP32]
Hi everyone. I've written what I hope will be a useful resource for the community. https://www.doctormonk.com/2024/06/new-book-on-esp32-and-micropython.html

**Comment by andypiper** [comment]
I know this will be a worthwhile addition to our knowledge, so I've just [submitted a PR](https://github.com/mcauser/awesome-micropython/pull/152) to the [Awesome MicroPython](https://awesome-micropython.com) list 👍🏻

**Comment by simonmonk** [comment]
Thank you very much Andy!

Si.

On Tue, 11 Jun 2024 at 10:32, Andy Piper ***@***.***> wrote:

> I know this will be a worthwhile addition to our knowledge, so I've just submitted
> a PR <https://github.com/mcauser/awesome-micropython/pull/152> to the Awesome
> MicroPython <https://awesome-micropython.com> list 👍🏻
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/micropython/discussions/15255#discussioncomment-9736862>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAQC6UOVO7OC7TG4EHM4LFDZG27ZZAVCNFSM6AAAAABJDYS7TCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOMZWHA3DE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
--
Dr. Simon Monk
+44 7544 373 418
http://simonmonk.org
@simonmonk2
***@***.***@floss.social​
